crates/revm/src/instructions.rs

```diff
@@ -86,9 +86,9 @@ pub fn eval<H: Host, S: Spec>(opcode: u8, interp: &mut Interpreter, host: &mut H
         246_u8..=249_u8 => Return::OpcodeNotFound,
         251_u8..=252_u8 => Return::OpcodeNotFound,*/
         opcode::STOP => Return::Stop,
-        opcode::ADD => op2_u256_tuple!(interp, overflowing_add),
-        opcode::MUL => op2_u256_tuple!(interp, overflowing_mul),
```

Comment:
Computing whether a mul overflows is actually a surprising amount of extra work. Since we don't need that here, better to use the wrapping version. (No extra cost for add/sub, but changing those too for symmetry and to remove all uses of `op2_u256_tuple`.)